Assets/Scripts/Scenes/InGame/Player/PlayerGUIPresenter.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using System.Collections.Generic;
+using UniRx;
+using UnityEngine;
+
+namespace Scenes.Ingame.Player
+{
+    /// <summary>
+    /// PlayerStatus��DisplayPlayerStatusManager�̋��n�����s���N���X
+    /// MV(R)P�ɂ�����Presenter�̖�����z��
+    /// </summary>
+    public class PlayerGUIPresenter : MonoBehaviour
+    {
+        //model
+        [SerializeField] private PlayerStatus[] _playerStatuses;
+        //View
+        [SerializeField] private DisplayPlayerStatusManager _displayPlayerStatusManager;
+
+
+        // Start is called before the first frame update
+        void Awake()
+        {
+            _displayPlayerStatusManager.OnCompleteSort
+                .Subscribe(_ => 
```

Comment:
```suggestion
            _displayPlayerStatusManager.OnCompleteSort
                .FirstOrDefault()
                .Subscribe(_ => 
``` 
1度しかイベントは発行されない設計ですが、念のためにSubscribe側でも1度しか受け取らないようにしておくと安全です。
フィルタリング処理は面白いので、今後使う際は見ておくと良いかもしれません。
参考：https://light11.hatenadiary.com/entry/2018/11/17/150424